pennylane/tape/qnode.py

```diff
@@ -685,14 +725,15 @@ def qfunc_decorator(func):
     return qfunc_decorator
 
 
-def draw(_qnode, charset="unicode"):
+def draw(_qnode, charset="unicode", wire_order=None):

```

Comment:
Nope, this is a high level function `qml.draw()`, so its signature doesn't need to match the methods! And the kwargs in those cases is superfluous anyway, and only exists so that the tests pass in both modes. It can be deleted as soon as tape is default